index.rst

```diff
@@ -0,0 +1,23 @@
+.. Mbed TLS documentation master file, created by
+   sphinx-quickstart on Thu Jul 29 17:27:29 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Mbed TLS's documentation!
+====================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
```

Comment:
Before this can become the home page of Mbed TLS on the web, this needs some important bits of information:

* What this site is (a place for documentation).
* What this site isn't (finished).
* Where to find what isn't on this site (github, trustedfirmware, old website).